docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3'
```

Comment:
Would it be better to name this file `docker-compose.dev.yml` to indicate that it's not intended for production?

Reply:
ah, that would make sense indeed.